certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+  "basePath": "/api/v2.1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/account/{AccountId}/callback": {
+      "x-ms-notification-content": {
+        "description": "New Incoming SMS",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "AccountName": {
+              "type": "string",
+              "description": "The name of the account the callback is associated to",
+              "title": "Account Name"
+            },
+            "ChannelCode": {
+              "type": "string",
+              "description": "Code of the channel the message has been sent over (the actual phone number for long codes)",
+              "title": "Channel Code"
+            },
+            "ChannelId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ID of the channel the message has been sent over",
+              "title": "ChannelId"
+            },
+            "ContactId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Contact ID if any (null for unknown contacts)",
+              "title": "ContactId"
+            },
+            "Message": {
+              "type": "string",
+              "description": "The actual message text",
+              "title": "Message"
+            },
+            "PhoneNumber": {
+              "type": "string",
+              "description": "Target phone number",
+              "title": "PhoneNumber"
+            },
+            "AccountId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "The ID of the account the callback is associated to",
+              "title": "AccountId"
+            },
+            "CallbackTimestamp": {
+              "type": "string",
+              "description": "Date and time the callback has been sent. Format: yyyy-MM-ddTHH:mm:ss",
+              "title": "Callback Timestamp"
+            },
+            "CallbackToken": {
+              "type": "string",
+              "description": "CallbackToken"
+            },
+            "CallbackType": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Always 11 for Incoming Message Callback",
+              "title": "CallbackType",
+              "x-ms-visibility": "internal"
+            },
+            "CallbackURL": {
+              "type": "string",
+              "description": "URL this callback has been sent to",
+              "title": "CallbackURL",
+              "x-ms-visibility": "internal"
+            },
+            "TransactionId": {
+              "type": "string",
+              "description": "Unique token",
+              "title": "TransactionId"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Incoming messages callback created"
```

Comment:
@TrueDialogDynamics Can you please let us know if this post action has schema definition?

Reply:
@sriyen-msft  Added schema for all the responses in [this commit](https://github.com/microsoft/PowerPlatformConnectors/pull/1337/commits/d4d6d4c7b8aa28c0eceb87f9eccd4625ec7c5d84)